Documentation/ApiOverview/SymfonyExpressionLanguage/Index.rst

```diff
@@ -44,16 +44,18 @@ Registering new provider within an extension
 There has to be a provider, no matter whether variables or functions will be provided.
 
 The provider is registered in the extension file :file:`/Configuration/ExpressionLanguage.php`, depending on
-the extension's custom PHP class name:
+the extension's custom PHP class name::
```

Comment:
Can you do the same fix in the following section? (I cannot make suggestions on Github for unchanged code.)